webdriver-spec.html

```diff
@@ -2261,6 +2273,14 @@
       <p>Otherwise, set the "<code>proxy</code>" entry
        in <var>matched capabilities</var> to <var>capability value</var>.
 
+     <dt>"<code>setWindowRect</code>"
+     <dd><p>If <var>capability value</var> is not a boolean
```

Comment:
Since we can't really influence if it can be done that way or not we should just ignore what the user puts here